web/ui/dashboard/src/app/private/pages/settings/personal-keys/personal-keys.component.html

```diff
@@ -0,0 +1,114 @@
+<!-- loader  -->
+<ng-container *ngIf="isFetchingKeys && !showCreateNewTokenForm">
+	<div class="flex justify-between items-center py-12px" *ngFor="let index of loaderIndex">
+		<div>
+			<div class="flex items-center mb-8px">
+				<convoy-skeleton-loader className="h-20px w-100px"></convoy-skeleton-loader>
+				<convoy-skeleton-loader className="h-14px w-60px ml-10px"></convoy-skeleton-loader>
+			</div>
+			<convoy-skeleton-loader className="h-14px w-120px"></convoy-skeleton-loader>
+		</div>
+		<convoy-skeleton-loader className="h-14px w-120px"></convoy-skeleton-loader>
+	</div>
+</ng-container>
+
+<!-- personal access tken header  -->
+<div class="flex justify-between items-center mb-28px" *ngIf="!showCreateNewTokenForm && personalAccessKeys?.content?.length > 0">
```

Comment:
@Oluwadaminiola This page should be under User Settings not Organisation Settings.